src/installer/corehost/cli/hostpolicy/hostpolicy_context.cpp

```diff
@@ -114,15 +114,38 @@ int hostpolicy_context_t::initialize(hostpolicy_init_t &hostpolicy_init, const a
     fx_definition_vector_t::iterator fx_end;
     resolver.get_app_fx_definition_range(&fx_begin, &fx_end);
 
+    pal::string_t resolved_frameworks;
     pal::string_t app_context_deps_str;
-    fx_definition_vector_t::iterator fx_curr = fx_begin;
-    while (fx_curr != fx_end)
+    bool is_app = host_mode != host_mode_t::libhost;
```

Comment:
Doesn't the first element in `fx_definitions` represent the app even in libhost mode?